closed/OpenJ9.gmk

```diff
@@ -341,6 +341,7 @@ ifeq (true,$(OPENJ9_ENABLE_CMAKE))
 	-DOPENJ9_BUILD=true \
 	-DOPENJ9_SHA=$(OPENJ9_SHA) \
 	-DOPENJDK_VERSION_NUMBER_FOUR_POSITIONS=$(VERSION_NUMBER_FOUR_POSITIONS) \
+	-DOMR_MIXED_REFERENCES_MODE="$(OMR_MIXED_REFERENCES_MODE)" \
```

Comment:
Please move this up to retain sorted order of variable definitions (after `OMR_DDR`).
I don't think the quotes are necessary (the value will be one of `off`, `static` or `dynamic`).